README.md

```diff
@@ -84,6 +84,7 @@ and a transform stream to decompress the file.
 | [options.check] | <code>Boolean</code> | <code>false</code> | enable write check |
 | [options.bmap] | <code>String</code> |  | bmap file contents |
 | [options.bytesToZeroOutFromTheBeginning] | <code>Number</code> |  | bytes to zero out from the beginning (bmap only) |
+| [options.checksumAlgorithms] | <code>Array.&lt;String&gt;</code> |  | checksums to calculate |
```

Comment:
Since this is passed as an Array, what happens if you ask for the same checksum more than once? Does the array get internally converted into the equivalent of a set, or would it (redundantly) calculate the same checksum more than once?